features/media-fix-orientation.feature

```diff
@@ -0,0 +1,111 @@
+Feature: Fix WordPress attachments orientation
+
+  Background:
+    Given a WP install
+
+  @require-wp-4.0
+  Scenario: Try to fix orientation for all images while none exists
+    When I try `wp media fix-orientation`
+    Then STDERR should be:
+      """
+      Error: No images found.
+      """
+
+  @require-extension-exif @require-wp-4.0
+  Scenario: Fix orientation for all images
+    Given download:
+      | path                             | url                                                                            |
+      | {CACHE_DIR}/landscape-2.jpg      | https://raw.githubusercontent.com/thrijith/test-images/master/Landscape_2.jpg  |
+      | {CACHE_DIR}/landscape-5.jpg      | https://raw.githubusercontent.com/thrijith/test-images/master/Landscape_5.jpg  |
+      | {CACHE_DIR}/portrait-4.jpg       | https://raw.githubusercontent.com/thrijith/test-images/master/Portrait_4.jpg   |
+      | {CACHE_DIR}/portrait-6.jpg       | https://raw.githubusercontent.com/thrijith/test-images/master/Portrait_6.jpg   |
+    And I run `wp option update uploads_use_yearmonth_folders 0`
+
+    When I run `wp media import {CACHE_DIR}/landscape-2.jpg --title="Landscape Two" --porcelain`
+    Then save STDOUT as {LANDSCAPE_TWO}
+
+    When I run `wp media import {CACHE_DIR}/landscape-5.jpg --title="Landscape Five" --porcelain`
+    Then save STDOUT as {LANDSCAPE_FIVE}
+
+    When I run `wp media import {CACHE_DIR}/portrait-4.jpg --title="Portrait Four" --porcelain`
+    Then save STDOUT as {PORTRAIT_FOUR}
+
+    When I run `wp media fix-orientation --dry-run`
+    Then STDOUT should contain:
+    """
+    3 "Portrait Four" (ID {PORTRAIT_FOUR}) will be affected.
+    """
+
+    And STDOUT should contain:
+    """
+    3 "Landscape Five" (ID {LANDSCAPE_FIVE}) will be affected.
+    """
+
+    And STDOUT should contain:
+    """
+    3 "Landscape Two" (ID {LANDSCAPE_TWO}) will be affected.
+    Success: 3 of 3 images will be affected.
```

Comment:
You need to separate these two, otherwise the tests will break if "Landscape Two" is not the last item.